### DISCUSSION TITLE
SecurlyKill suddenly not working on new chromebook

### DISCUSSION THREAD

**Discussion by Winterfyre** [3kh0/ext-remover#895] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
So I've been using the combo of Ublock to run javascript and then the SecurlyKill code really effectively for about 5 months, but when that chromebook broke, I had to get a new one. Suddenly, the SecurlyKill code seems to be not working on the last step, as I can still run the code through the Ublock prompt, it pulls up the button and displays the text once clicked, but it no longer actually unblocks anything. I've double-triple checked that I'm using the right code, and nothing has changed from before other than a different chromebook and I think an update or two. Any fixes y'all know of?

**Comment by S-PScripts** [comment]
Just one update is enough for the chromeOS devs to fix a bug :(
